### PR TITLE
Implement byron vote creation and submission

### PR DESCRIPTION
**Pull request by Jimbo4350**
Issue
-----------

- #803 

- This PR **does not result** in breaking changes to upstream dependencies.

Checklist
---------
- [x] This PR contains all the work required to resolve the linked issue.

- [x] The work contained has sufficient documentation to describe what it does and how to do it.

- [x] I have committed clear and descriptive commits. Be considerate as somebody else will have to read these.

- [x] I have added the appropriate labels to this PR.

### CODE REVIEW

**Comment by Jimbo4350** [comment]
I will update how the bryon commands are grouped and structured in a follow up PR so that QA and DevOps can begin to try out creation/submission of update proposals and votes.

File: README.md [hunk 1]
```diff
@@ -333,7 +335,9 @@ You can also check your proposal's validity using the [`validate-cbor`](#validat
```
Comment (ArturWieczorek) [inline README.md:335]: Just a suggestion:

If we are giving examples for creating and submitting proposal vote then maybe we can also add similar examples for update proposals with mandatory arguments: 

cardano-cli -- byron create-byron-update-proposal --config configuration/defaults/liveview/config-0.yaml --signing-key configuration/defaults/liveview/genesis/delegate-keys.000.key --protocol-version-major 1 --protocol-version-minor 0 --protocol-version-alt 0 --application-name app --software-version-num 1 --system-tag linux --installer-hash da39a3ee5e6b4b0d3255bfef95601890afd80701 --filepath ProtocolUpdateProposalFile 

and 

cardano-cli -- byron submit-byron-update-proposal --config configuration/defaults/liveview/config-0.yaml --filepath ProtocolUpdateProposalFile --socket-path socket/node-0-socket 

with example output that user should expect
Update proposal TxId: upid:b982f442

File: README.md [hunk 2]
```diff
@@ -348,6 +352,29 @@ The socket path  must either be specified as an argument (`--socket-path`) or sp
...
+                        --config configuration/defaults/liveview/config-0.yaml
+                        --signing-key configuration/defaults/liveview/genesis/delegate-keys.000.key
+                        --proposal-filepath ProtocolUpdateProposalFile
+                        --vote-yes
+                        --output-filepath UpdateProposalVoteFile
+```
+
+Byron vote submission:
+
+```cabal exec cardano-cli -- byron submit-byron-proposal-vote
```
Comment (ArturWieczorek) [inline README.md:372]: New line to fix formatting issue

**Comment by Jimbo4350** [comment]
bors merge

**Pull request merged**